packages/flutter/lib/src/material/search_anchor.dart

```diff
@@ -982,6 +983,21 @@ class SearchController extends TextEditingController {
 /// the search bar. Typically only one or two action icons are included.
 /// These actions can represent additional modes of searching (like voice search),
 /// a separate high-level action (such as current location) or an overflow menu.
+///
+/// {@tool dartpad}
+/// This example shows how to use a [SearchBar] as the builder of the [SearchAnchor].
```

Comment:
SearchBar isn't a SearchAnchorChildBuilder

Reply:
Ah right, it should be the builder's return value instead of the builder. Just updated.